examples/prom/elliptic_eigenproblem_global_rom.cpp

```diff
@@ -0,0 +1,939 @@
+/******************************************************************************
+ *
+ * Copyright (c) 2013-2024, Lawrence Livermore National Security, LLC
+ * and other libROM project developers. See the top-level COPYRIGHT
+ * file for details.
+ *
+ * SPDX-License-Identifier: (Apache-2.0 OR MIT)
+ *
+ *****************************************************************************/
+
+//                       libROM MFEM Example: Elliptic Eigenproblem (adapted from ex11p.cpp)
+//
+// =================================================================================
+//
+// Description:  This example code demonstrates the use of MFEM and libROM to
+//               define a simple projection-based reduced order model of the
+//               eigenvalue problem -Delta ((1 + alpha v) u) = lambda u with homogeneous
+//               Dirichlet boundary conditions where alpha is a scalar ROM parameter
+//               controlling the frequency of v.
+//
+//               We compute a number of the lowest eigenmodes by discretizing
+//               the Laplacian and Mass operators using a FE space of the
+//               specified order, or an isoparametric/isogeometric space if
+//               order < 1 (quadratic for quadratic curvilinear mesh, NURBS for
+//               NURBS mesh, etc.)
+//
+// Offline phase: elliptic_eigenproblem_global_rom -offline -p 2 -id 0 -a 0.8
+//                elliptic_eigenproblem_global_rom -offline -p 2 -id 1 -a 0.9
+//                elliptic_eigenproblem_global_rom -offline -p 2 -id 2 -a 1.1
+//                elliptic_eigenproblem_global_rom -offline -p 2 -id 3 -a 1.2
+//
+// Merge phase:   elliptic_eigenproblem_global_rom -merge -p 2 -ns 4
+//
+// FOM run (for error calculation):
+//                elliptic_eigenproblem_global_rom -fom -p 2 -f 1.0
```

Comment:
When you get a chance, would you please update the command line options, timing and error in the PR description and the header of the script? I think `-a 1.1` and `-a 1.2` in the offline phase do not work anymore, and it might be `-a` instead of `-f` in the online phase.

Reply:
Update: I saw that the new command lines and results in PR description, and the new results in the header. It remains to correct the command lines in the header.